setup.py

```diff
@@ -1,6 +1,15 @@
 #!/usr/bin/env python
 import glob
 from setuptools import setup, Extension
+from Cython.Build import cythonize
```

Comment:
What do you think about doing this import only if is_pypy == True? This way Cython won't be required for CPython users.